defender/connection.py

```diff
@@ -19,7 +19,15 @@ def get_redis_connection():
     if config.MOCK_REDIS:  # pragma: no cover
         return mocked_redis  # pragma: no cover
     else:  # pragma: no cover
-        redis_config = parse_redis_url(config.DEFENDER_REDIS_URL)
+        if config.DEFENDER_REDIS_URL:
+            redis_config = parse_redis_url(config.DEFENDER_REDIS_URL)
+        else:
+            redis_config = {
+                "DB": 0,
+                "PASSWORD": None,
+                "HOST": config.REDIS_HOST,
+                "PORT": 6379,
+            }
```

Comment:
Please can you update config.py (proxy settings file between django-defender and bcore settings) with config.
REDIS_HOST because REDIS_HOST doesn't exist there.